account_invoice_base_tax_zero/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2019 Alessandro Camilli - <alessandrocamilli@openforce.it>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Account Invoice Base Tax Zero',
+    'summary': 'Generate account move lines including tax lines with amount 0',
+    'version': '11.0.1.0.0',
+    'license': 'AGPL-3',
+    'category': 'Accounting',
+    'author': 'Openforce,Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/account-invoicing/',
+    'depends': ['account'],
+    'data': [],
```

Comment:
@alessandrocamilli IMO if not any file thne remove it

Reply:
line 'data' removed